docs/AboutUs.md

```diff
@@ -38,12 +38,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Data
 
-### Jean Doe
+### Chong Jun Wei
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/junwei.png" width="200px">
```

Comment:
Image name should also follow your github username in simple letters! (It won't get tracked for your progress otherwise)